users/router.js

```diff
@@ -1,7 +1,6 @@
 var express = require('express');
 var mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost:80085');
-var userModels = require('./userModels');
+mongoose.connect('mongodb://localhost:27017');
```

Comment:
This is no longer needed as it is handled in `server.js` 
Merge `master` into your branch to get the latest changes